security/pkg/nodeagent/cache/secretcache_test.go

```diff
@@ -135,7 +135,7 @@ func (u *UpdateTracker) Expect(want map[string]int) {
 			return fmt.Errorf("wanted %+v got %+v", want, u.hits)
 		}
 		return nil
-	}, retry.Timeout(time.Second*2), retry.Delay(time.Millisecond))
+	}, retry.Timeout(time.Second*5), retry.Delay(time.Millisecond*500))
```

Comment:
```suggestion
	}, retry.Timeout(time.Second*5))
```

Much faster and still just as reliable. This uses exponential backoff starting at 10ms